code/modules/vehicles/ridden.dm

```diff
@@ -63,12 +64,13 @@
 		return FALSE
 	if(legs_required)
 		var/how_many_legs = user.get_num_legs()
-		if(how_many_legs < legs_required)
+		if(how_many_legs < legs_required && message_cooldown < world.time)

```

Comment:
bad.

only add the check for the to_chat, otherwise you'll allow them to do the thing if the message is on cooldown.